refs/types.go

```diff
@@ -183,6 +188,43 @@ func (s *SubnetID) GetValue() uint32 {
 	return 0
 }
 
+// MarshalText encodes SubnetID into text format according to NeoFS API V2 protocol:
+// value in base-10 integer string format.
+//
+// Implements encoding.TextMarshaler.
+func (s *SubnetID) MarshalText() ([]byte, error) {
+	num := s.GetValue() // NPE safe, returns zero on nil (zero subnet)
+
+	return []byte(strconv.FormatUint(uint64(num), 10)), nil
```

Comment:
why `uint64`?

Reply:
`num` is of type `uint32`, `strconv.FormatUint`accepts `uint64`.